src/components/loading/Loading.vue

```diff
@@ -116,6 +117,7 @@ export default {
         if (typeof window !== 'undefined') {
             document.removeEventListener('keyup', this.keyPress)
         }
+        clearTimeout(this.timeOutID)
```

Comment:
@Arooba-git This should not be necessary. Because `timeOutID` is set only if `Loading` has programmatically been opened (`this.programmatic` is `true`) and programmatically opened `Loading` won't be destroyed until the timeout expires. Please see the line 83.

Reply:
@Arooba-git I have reverted the changes in your fork.